### DISCUSSION TITLE
How can i get arguments listed as in the shiny example, instead of having a table as done by default?

### DISCUSSION THREAD

**Discussion by JaumeAmoresDS** [machow/quartodoc#349] [Q&A] [answered]
Given a function:
```python
def myfunction (x: int, y: int=0):
   """Add x and y.

   Parameters
   ------------
     x: int
         First number
     y: int
         Second number
   Returns
   ---------
     Int: sum of x and y
   """
   return x+y
```

I would like the signature rendered as a list of arguments, something like:

x: First number (int)
y: Second number (int). Optional, default: 0.

Something like this is done in the shiny documentation example. I thought the trick was to use style: pkgdown, but it doesn't seem to be the case?

**Comment by machow** [comment, marked as answer]
This currently isn't possible without extending the renderer (as the shiny team does).

I have this PR starting the work to enable in quartodoc

https://github.com/machow/quartodoc/pull/325

And we have it as a priority to make this happen in the Great Tables docs, so I should be able to loop back and make it happen in the next month. (Sorry for the wait 😓 )

https://posit-dev.github.io/great-tables/reference/

**Reply by JaumeAmoresDS** [reply]
Thank you very much for your prompt response and for this amazing library :-)
